tests/test_decode.py

```diff
@@ -90,11 +90,11 @@ def testReadUInt(self):
                    b'\x15\x41\x42\x43']
         ints = [0x85, 0x4541, 0x254142, 0x15414243]
         
-        for id, i, numOut in zip(idBytes, range(1, len(idBytes) + 1), ints):
-            self.mockStream = StringIO(id)
+        for id, i, numOut in zip(idBytes, list(range(1, len(idBytes) + 1)), ints):
```

Comment:
I don't think `range()` needs to be cast to `list`.
Would it be any clearer to implement it like this?
```python
for i, (id, numOut) in enumerate(zip(idBytes, ints), 1):
```